install/Knative-custom-install.md

```diff
@@ -0,0 +1,222 @@
+| [`release.yaml`][8] ([knative/eventing-sources repo][8.1]) | - | - | Included       | -                 |
+| [`serving.yaml`][9]      | Included          | -               | -           | -                 |
+
+_*_ See [Installing observability plugins](../serving/installing-logging-metrics-traces.md)
+for details about the supported plugins and how to add monitoring, logging, and
+tracing to your cluster.
+
+[1]: https://github.com/knative/serving/releases/download/v0.2.2/release.yaml
+[1.1]: https://github.com/knative/serving/releases/tag/v0.2.2
+[2]: https://www.elastic.co/elk-stack
+[3]: https://github.com/knative/serving/releases/download/v0.2.2/release-no-mon.yaml
+[4]: https://github.com/knative/serving/releases/download/v0.2.2/release-lite.yaml
+[5]: ../serving/setting-up-a-logging-plugin.md
+[6]: https://github.com/knative/serving/releases/download/v0.2.2/build.yaml
+[6.1]: https://github.com/knative/build/releases/download/v0.2.0/release.yaml
+[6.2]: https://github.com/knative/build/releases/tag/v0.2.0
+[7]: https://github.com/knative/eventing/releases/download/v0.2.0/eventing.yaml
+[7.1]: https://github.com/knative/eventing/releases/tag/v0.2.0
+[8]: https://github.com/knative/eventing-sources/releases/download/v0.2.0/release.yaml
+[8.1]: https://github.com/knative/eventing-sources/releases/tag/v0.2.0
+[9]: https://github.com/knative/serving/releases/download/v0.2.2/serving.yaml
+
+Note: Each of the installable Knative resources are also listed in the Assets
+section of each Knative component's release page:
+* [Serving v0.2.2][1.1]
+* [Build v0.2.0][6.2]
+* [Eventing v0.2.0][7.1]
+
+### Installing Knative packages
+
+**Tip**: From the table above, you can copy and paste the URL and filename of
+the installation package that you want to install.
+
+1. To install a Knative package, run the `kubectl apply` command
+   and specify the package path and filename:
+
+   * To install an individual package:
+      ```bash
+      kubectl apply --filename [INSTALL_PACKAGE]
+      ```
+
+   * To install multiple packages, you can append additional `--filename [INSTALL_PACKAGE]`
+     flags to the `kubectl apply` command:
+      ```bash
+      kubectl apply --filename [INSTALL_PACKAGE] --filename [INSTALL_PACKAGE] \
+        --filename [INSTALL_PACKAGE]
+      ```
+
+    Where [`INSTALL_PACKAGE`] is the URL path and filename of the a Knative installation package:
+
+    `https://github.com/knative/[COMPONENT]/releases/download/[VERSION]/[FILENAME].yaml`
+
+    And `[COMPONENT]`, `[VERSION]`, and `[FILENAME]` are the Knative component, release version, and filename of the installable resource. For example:
+    * `https://github.com/knative/build/releases/download/v0.2.0/release.yaml`
+    * `https://github.com/knative/eventing/releases/download/v0.2.0/eventing.yaml`
+    * `https://github.com/knative/eventing-sources/releases/download/v0.2.0/release.yaml`
```

Comment:
What sources does this include?

Reply:
"core sources" as its referred to in the readme, which are k8s, github, and containers